README.md

```diff
@@ -1 +1,164 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **This is a replica of a portfolio from figma**
+- **So far I added a toolbar and headline**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> None
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+step 1:
+Clone this repo locally using git in the command line and the following command
+
+```
+git clone https://github.com/Jrmcuervo/hello-microverse.git
+```
+
+Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+Step 2:
+Copy All Project files and run using VSCODE. HTML can be viewed in browser.
+
+
+### Prerequisites
+
+In order to run this project you need:
+◦ Installation of Git
+◦ Installation of NPM
+◦ A working browser
+◦ Internet connection
+◦ A code editor like Visual Studio Code
+◦ Installation of Live Server plugging in Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```git   clone https://github.com/Jrmcuervo/replica-my-portfolio.git```
+
+### Install
+
+Install this project with:
+
+- No need of installation
+
+### Usage
+
+To run the project, open the it by launching the Live Server plugging in visual studio code or any other means that you know.
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+👤 **Ruben**
+
+- GitHub: [@Jrmcuervo](https://github.com/Jrmcuervo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **In the next part of the project I'm goin to add interactions to the navbar to show and hide the menu**
+- [ ] **Also I'm goin to build the next part of my portfolio that includes the projects and about me section**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
```

Comment:
[OPTIONAL] I think leaving incomplete statements on your readme makes it look less professional😉 I suggest you complete these incomplete statements. 🤗